include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -4063,121 +3701,118 @@ namespace internal
+        adjust_for_face_orientation(
+          dim,
+          n_components,
+          evaluation_flag,
+          &eval.get_orientation_map()(eval.get_face_orientation(), 0),
+          false,
+          eval.get_shape_info().n_q_points_face,
+          temp,
+          eval.begin_values(),
+          eval.begin_gradients(),
+          eval.begin_hessians());
+
+      return false;
+    }
 
-      (void)sm_ptr;
-
-      Processor<fe_degree, n_q_points_1d> p(n_components,
-                                            false,
-                                            src_ptr,
-                                            sm_ptr,
-                                            data,
-                                            dof_info,
-                                            values_quad,
-                                            gradients_quad,
-                                            hessians_quad,
-                                            scratch_data,
-                                            evaluate_values,
-                                            evaluate_gradients,
-                                            evaluate_hessians,
-                                            active_fe_index,
-                                            first_selected_component,
-                                            cells,
-                                            face_nos,
-                                            subface_index,
-                                            dof_access_index,
-                                            face_orientations,
-                                            orientation_map);
-
-      if (n_face_orientations == VectorizedArrayType::size())
-        return fe_face_evaluation_process_and_io<VectorizedArrayType::size()>(
-          p);
+    static bool
+    supports(
+      const EvaluationFlags::EvaluationFlags evaluation_flag,
+      const internal::MatrixFreeFunctions::ShapeInfo<VectorizedArrayType>
+        &                                                          shape_info,
+      const Number *                                               vector_ptr,
+      internal::MatrixFreeFunctions::DoFInfo::IndexStorageVariants storage)
+    {
+      const unsigned int fe_degree = shape_info.data[0].fe_degree;
+      if (fe_degree < 1 || !shape_info.data[0].nodal_at_cell_boundaries ||
+          (evaluation_flag & EvaluationFlags::gradients &&
+           (fe_degree < 2 ||
+            shape_info.data[0].element_type !=
+              MatrixFreeFunctions::tensor_symmetric_hermite)) ||
+          (evaluation_flag & EvaluationFlags::hessians) ||
+          vector_ptr == nullptr ||
+          shape_info.data[0].element_type >
+            MatrixFreeFunctions::tensor_symmetric ||
+          storage <
+            MatrixFreeFunctions::DoFInfo::IndexStorageVariants::contiguous)
+        return false;
       else
-        return fe_face_evaluation_process_and_io<1>(p);
+        return false;
```

Comment:
This will be switched to `true` once I have managed to clean up the `fe_face_evaluation_process_and_io` function.